nservicebus/upgrades/5to6.md

```diff
@@ -610,4 +610,38 @@ The BSON serializer build into the core has been removed. Use the [Newtonsoft se
 
 [MSMQ Subscription Authorization](/nservicebus/msmq/subscription-authorisation.md) is now done by the `SubscriptionAuthorizer` delegate at configuration time and not the `IAuthorizeSubscriptions` interface.
 
-snippet: 5to6-MsmqSubscriptionAuthorizer
\ No newline at end of file
+snippet: 5to6-MsmqSubscriptionAuthorizer
+
+## SQL Server Transport
```

Comment:
@SimonCropp what's the idea? Have a separate page in the upgrades directory for it and link to the "main" guide as related ?